src/features/layout/LeftNavBar.module.scss

```diff
@@ -42,5 +43,17 @@ ul.nav_list {
     .nav_desc {
       font-size: 14px;
     }
+
+    .nav_notifs {
+      background-color: use-color("error");
```

Comment:
I know that the notification is red today, but perhaps this is another good place to introduce our semantic coloring? That is, unseen notifications are not errors, more like info?